packages/grafana-data/src/field/getFieldDisplayValuesProxy.ts

```diff
@@ -44,12 +43,13 @@ export function getFieldDisplayValuesProxy(
         return undefined;
       }
       if (!field.display) {
+        throw new Error('Field missing display processor ' + field.name);
         // Lazy load the display processor
-        field.display = getDisplayProcessor({
-          field,
-          theme: options.theme,
-          timeZone: options.timeZone,
-        });
+        // field.display = getDisplayProcessor({
+        //   field,
+        //   theme: options.theme,
+        //   timeZone: options.timeZone,
+        // });
```

Comment:
Let's remove this! I hope this will not break anything - tested manually and haven't encountered anything problematic.